src/errors.test.ts

```diff
@@ -222,69 +222,71 @@ describe("errors.ts logError()", () => {
   it("should log regular Error instances", async () => {
     const errorMessage = "uh oh";
     const error = new Error(errorMessage);
-    const logMessage = "test message";
-    await logError(error, logMessage);
+    const logPrefix = "test message";
+    await logError(error, logPrefix);
 
-    sinon.assert.calledOnceWithExactly(loggerErrorSpy, `Error: ${logMessage} --> ${error}`, {
-      errorType: error.name,
-      errorMessage,
-      errorStack: error.stack,
-    });
+    sinon.assert.calledOnceWithExactly(
+      loggerErrorSpy,
+      `[${logPrefix}] error: ${error.name}: ${errorMessage}`,
+      {
+        errorType: error.name,
+        errorMessage,
+        errorStack: error.stack,
+      },
+    );
```

Comment:
Looks like this reverts some of the changes from https://github.com/confluentinc/vscode/pull/1453 - does this branch need to get rebased against `main`?